views/js/qtiCreator/renderers/Img.js

```diff
@@ -19,17 +19,22 @@
 define([
     'lodash',
     'taoQtiItem/qtiCommonRenderer/renderers/Img',
-    'taoQtiItem/qtiCreator/widgets/static/img/Widget'
-], function(_, Renderer, Widget){
+    'taoQtiItem/qtiCreator/widgets/static/img/Widget',
+    'taoQtiItem/qtiCreator/widgets/static/figure/Widget',
+    'taoQtiItem/qtiCreator/model/Figure',
+    'taoQtiItem/qtiCreator/helper/findParentElement'
+], function (_, Renderer, Widget, FigureWidget, FigureModel, findParentElement){
     'use strict';
 
     const CreatorImg = _.clone(Renderer);
 
-    CreatorImg.render = function(img, options){
-
+    CreatorImg.render = function (img, options){
         const $container = Renderer.getContainer(img);
-        if ($container.parent('figure').length) {
+        if ($container.parent('figure').length || $container.parent('span').length && $container.parent('span').data('figure')) {
```

Comment:
There is a discrepancy between this and original PR in this line, is this expected? below original PR merged changes.
```
Renderer.getContainer(img);
        if ($container.parent('figure').length || $container.parent('span').length && $container.parent('span').data('serial') && $container.parent('span').data('serial').includes('figure')) {
```

Reply:
Here backport for 2 PRs, this changes from second https://github.com/oat-sa/extension-tao-itemqti/pull/2291